### PR TITLE
Remove hr

### PR DESCRIPTION
**Pull request by bjedrzejewski**
Now it is using css. Removing css styling from js is separate issue.

### CODE REVIEW

File: src/sidebars/favourites/favourites.less [hunk 1]
```diff
@@ -4,7 +4,7 @@
 @favourite-padding-right: 5px;
 @favourite-padding-bottom: 10px;
 @top-height: 62px;
-@total-height: (@favourite-height + @splitter-height);
+@total-height: (@favourite-height);
```
Comment (owennw) [inline src/sidebars/favourites/favourites.less]: Do we need this variable?

File: src/sidebars/favourites/favourites.less [hunk 2]
```diff
@@ -21,6 +26,7 @@
   padding-left: @favourite-padding-left;
   cursor: pointer;
   height: @favourite-height;
+  border-bottom: 1px solid @icon-colour;
```
Comment (owennw) [inline src/sidebars/favourites/favourites.less]: We can reuse this `1px solid @icon-colour` as a variable.

File: src/sidebars/search/search.less [hunk 3]
```diff
@@ -9,9 +9,15 @@
...
 
+
+.search .sidetab:first-of-type{
+  border-top: 1px solid @icon-colour;
+}
+
 .search-result {
   height: @search-result-height;
   cursor: pointer;
+  border-bottom: 1px solid @icon-colour;
```
Comment (owennw) [inline src/sidebars/search/search.less]: Same as above!

**Comment by bjedrzejewski** [comment]
@owennw i implemented the fixes, ready to merge

**Pull request merged**